{"13.0.1.0.1":{"info":{"author":"ForgeFlow, Odoo Community Association (OCA)","author_email":"support@odoo-community.org","bugtrack_url":null,"classifiers":["Framework :: Odoo","License :: OSI Approved :: GNU Affero General Public License v3","Programming Language :: Python"],"description_content_type":"","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/OCA/account-invoicing","keywords":"","license":"AGPL-3","maintainer":"","maintainer_email":"","name":"odoo13-addon-account-move-exception","package_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","platform":"","project_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","project_urls":{"Homepage":"https://github.com/OCA/account-invoicing"},"provides_extra":null,"release_url":"https://pypi.org/project/odoo13-addon-account-move-exception/13.0.1.0.1/","requires_dist":["odoo13-addon-base-exception","odoo (<13.1dev,>=13.0a)"],"requires_python":">=3.5","summary":"Custom exceptions on account move","version":"13.0.1.0.1","yanked":false,"yanked_reason":null},"last_serial":19574735,"urls":[{"comment_text":"","digests":{"blake2b_256":"44034173c3d55605077c5766fc292ee5c93878bd8935396926e5e008c3b6e374","md5":"a8ba8ccdb33b849b86eed183442b9280","sha256":"cba5ba46a7f5f92b9bd663729b6e43c154d82985289cf2c68cbdd85faa30a28c"},"downloads":-1,"filename":"odoo13_addon_account_move_exception-13.0.1.0.1-py3-none-any.whl","has_sig":false,"md5_digest":"a8ba8ccdb33b849b86eed183442b9280","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.5","size":29082,"upload_time":"2021-05-15T05:21:39","upload_time_iso_8601":"2021-05-15T05:21:39.553554Z","url":"https://files.pythonhosted.org/packages/44/03/4173c3d55605077c5766fc292ee5c93878bd8935396926e5e008c3b6e374/odoo13_addon_account_move_exception-13.0.1.0.1-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"13.0.1.0.1.dev2":{"info":{"author":"ForgeFlow, Odoo Community Association (OCA)","author_email":"support@odoo-community.org","bugtrack_url":null,"classifiers":["Framework :: Odoo","License :: OSI Approved :: GNU Affero General Public License v3","Programming Language :: Python"],"description_content_type":"","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/OCA/account-invoicing","keywords":"","license":"AGPL-3","maintainer":"","maintainer_email":"","name":"odoo13-addon-account-move-exception","package_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","platform":"","project_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","project_urls":{"Homepage":"https://github.com/OCA/account-invoicing"},"provides_extra":null,"release_url":"https://pypi.org/project/odoo13-addon-account-move-exception/13.0.1.0.1.dev2/","requires_dist":["odoo13-addon-base-exception","odoo (<13.1dev,>=13.0a)"],"requires_python":">=3.5","summary":"Custom exceptions on account move","version":"13.0.1.0.1.dev2","yanked":false,"yanked_reason":null},"last_serial":19574735,"urls":[{"comment_text":"","digests":{"blake2b_256":"d3e69ab03ea87e68e9a485adf0259cb33780b348184ce6386a09a0345253e093","md5":"aac13730cccaa89f80102e8d2cbe8753","sha256":"fa4b1e55eca385cf166b16a3465ac24205a27e5a0b6c291ab92c5295c863662b"},"downloads":-1,"filename":"odoo13_addon_account_move_exception-13.0.1.0.1.dev2-py3-none-any.whl","has_sig":false,"md5_digest":"aac13730cccaa89f80102e8d2cbe8753","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.5","size":29189,"upload_time":"2021-04-30T05:20:50","upload_time_iso_8601":"2021-04-30T05:20:50.714727Z","url":"https://files.pythonhosted.org/packages/d3/e6/9ab03ea87e68e9a485adf0259cb33780b348184ce6386a09a0345253e093/odoo13_addon_account_move_exception-13.0.1.0.1.dev2-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"13.0.1.0.2.dev1":{"info":{"author":"ForgeFlow, Odoo Community Association (OCA)","author_email":"support@odoo-community.org","bugtrack_url":null,"classifiers":["Framework :: Odoo","Framework :: Odoo :: 13.0","License :: OSI Approved :: GNU Affero General Public License v3","Programming Language :: Python"],"description_content_type":"","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/OCA/account-invoicing","keywords":"","license":"AGPL-3","maintainer":"","maintainer_email":"","name":"odoo13-addon-account-move-exception","package_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","platform":null,"project_url":"https://pypi.org/project/odoo13-addon-account-move-exception/","project_urls":{"Homepage":"https://github.com/OCA/account-invoicing"},"provides_extra":null,"release_url":"https://pypi.org/project/odoo13-addon-account-move-exception/13.0.1.0.2.dev1/","requires_dist":["odoo13-addon-base-exception","odoo <13.1dev,>=13.0a"],"requires_python":">=3.5","summary":"Custom exceptions on account move","version":"13.0.1.0.2.dev1","yanked":false,"yanked_reason":null},"last_serial":19574735,"urls":[{"comment_text":"","digests":{"blake2b_256":"1f967811486a8cfb11dd5fb7c590f40d83f1d4b52a6e6bd6c7c3c633c77c6aba","md5":"4acf02942832b116813fb3a663a2237d","sha256":"86173938b2ec00e96f7148f2d0147c88efe120add0b2c9e129bccf2489d7f9c0"},"downloads":-1,"filename":"odoo13_addon_account_move_exception-13.0.1.0.2.dev1-py3-none-any.whl","has_sig":false,"md5_digest":"4acf02942832b116813fb3a663a2237d","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.5","size":29458,"upload_time":"2023-09-03T11:31:30","upload_time_iso_8601":"2023-09-03T11:31:30.639229Z","url":"https://files.pythonhosted.org/packages/1f/96/7811486a8cfb11dd5fb7c590f40d83f1d4b52a6e6bd6c7c3c633c77c6aba/odoo13_addon_account_move_exception-13.0.1.0.2.dev1-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}